src/operator/subgraph/mkldnn/mkldnn_fc_property.h

```diff
@@ -113,13 +113,17 @@ class SgMKLDNNFCSelector : public SubgraphSelector {
           }
         }
         if (!quantized_ && (new_node.op() == Op::Get("square") ||
+            new_node.op() == Op::Get("_npi_square") ||
             new_node.op() == Op::Get("sqrt") ||
-            new_node.op() == Op::Get("exp"))) {
+            new_node.op() == Op::Get("_npi_sqrt") ||
+            new_node.op() == Op::Get("exp") ||
+            new_node.op() == Op::Get("_npi_exp"))) {
           matched_list_.push_back(&new_node);
           status_ = kSuccess;
           return true;
         }
-        if (new_node.op() == Op::Get("abs")) {
+        if (new_node.op() == Op::Get("abs") ||
+            new_node.op() == Op::Get("_npi_absolute")) {
```

Comment:
If you will be changing something I think it would be nice to put those checks in pairs e.g. `if (new_node.op() == Op::Get("abs") || new_node.op() == Op::Get("_npi_absolute")) {` or `new_node.op() == Op::Get("exp") || new_node.op() == Op::Get("_npi_exp")`.

Reply:
Like this? 
![image](https://user-images.githubusercontent.com/17185347/123627075-dc7aa180-d811-11eb-9437-53f80df9af31.png)

Reply:
Yes, I was thinking about something similar to this. That was just a suggestion so if you do not like it, feel free to ignore it :)

Reply:
ok thank you it was worth a try :)